exercise-1.js

```diff
@@ -444,74 +711,148 @@
+        //and create a doMath() function that randomly selects one of the mathematical operations whenever it is invoked.  
+        //The doMath() function should print to the console the mathetmatical function that was carried out.  The doMath() function 
+        //should return the computed value of any operation performed.
 
-        // ex : 2,3 => doMath(2,3) => adding : 5
-        const operations = [];
-        function doMath(x,y) {};
+        // ex : 2,3 => doMath(2,3) => adding : 5       
 
         //your code...
-
-
-
+        const add = function(x, y) {return x + y;};
+        const subtract = function(x, y) {return x - y;};
+        const multiply = function(x, y) {return x * y;};
+        const divides = function(x, y) {return x / y;};
+        const operations = [add, subtract, multiply, divides];
+        function doMath(x,y) {
+            const randomElement = operations[Math.floor(Math.random() * operations.length)];
+            let val = randomElement(x,y);
+            console.log(`${randomElement} ${val}`);
+        };
+        doMath(5,2);
         /************************************************************* */
-        //- Create a Higher Order Function called multiple(x) that takes a single parameter.  This HOF should return another function fn(y) that accepts another single parameter y.  This inner function should compute the product of it's parameter with the parameter passed into multiple.  Use this returned "first-class" function to compute triples of any given number.
+        //- Create a Higher Order Function called multiple(x) that takes a single parameter.  
+        //This HOF should return another function fn(y) that accepts another single parameter y.  
+        //This inner function should compute the product of it's parameter with the parameter passed into multiple.  
+        //Use this returned "first-class" function to compute triples of any given number.
 
         //your code...
-
-
-        //- Write an outer function called stockGain that has cost basis (basis) as a parameter; declare a variable called message that holds " is how much the stock has increased".  Return an inner function with years (yrs) as a parameter and declare a variable for growth rate (r) of 5%. Console log your calculation.
-
-        //your code
-
-
-        // Once finished, declare a variable called futureValue that holds your stockGain function and enter any amount for the cost basis and a number for the number of years.  Run the function returned by the higher order function to display the future value of the stock.  
+        
+            function multiple(x) {
+                let result = fn(5);
+                function fn(y) {
+                    let value = x * y;
+                    return value
+                }
+                return result;
+            }
+        
+        let result = multiple(5);
+        console.log(result);
```

Comment:
This implementation doesn't behave as expected, and the use of the higher-order function doesn't meet the requirement.